core/templates/dev/head/pages/review-test-page/review-test-page.directive.ts

```diff
@@ -72,6 +73,8 @@ oppia.directive('reviewTestPage', ['UrlInterpolationService', function(
           $http.get(reviewTestsDataUrl).then(function(result) {
             var skillIdList = [];
             var skillDescriptions = [];
+            PageTitleService.setPageTitle(
```

Comment:
@Jamesjay4199  What will be the page title after page loads and before the `reviewTestsDataUrl` is fetched? If it's nothing, then should we initialize the PageTitleService with some default value (maybe just `Oppia`)? (This can be resolved in a separate PR if required)

Reply:
Ignore my last comment: I default page title it will be `Oppia` ([ref](https://github.com/oppia/oppia/pull/7187/files#diff-d29ec6dc0e3559f25af84bae0165efd7R4)).